extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -920,7 +920,7 @@ describe('empty Slides functional', () => {
     }).to.throw(/should have at least 1 slide/);
     expect(() => {
       slides.layoutCallback();
-    }).to.not.throw();
+    }).to.throw(/Element expected/);
```

Comment:
Why do we have to assertElement [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/0.1/slides.js#L208)